staging/src/k8s.io/kubectl/pkg/cmd/run/run.go

```diff
@@ -173,6 +173,7 @@ func NewCmdRun(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Co
 
 func addRunFlags(cmd *cobra.Command, opt *RunOptions) {
 	cmdutil.AddDryRunFlag(cmd)
+	cmd.Flags().String("annotation", "", "Comma separated annotations to apply to the pod.")
```

Comment:
Why not make this a StringArray?

Reply:
I followed the labels flag for it, I'll change it